index.html

```diff
@@ -4,9 +4,29 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Python Jokes By Team J.L.J</title>
+  <link rel="stylesheet" href="styles/index.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Fira+Code&family=Inconsolata&family=Source+Code+Pro&display=swap" rel="stylesheet">
 </head>
 <body>
-  
+  <header>
+    <h1> Welcome to Python Jokes!</h1>
+    <!-- <h2 class="team"> Team Jenny X. & Laura C. & Jane K.</h2> -->
+  </header>
+    <nav class="main-nav">
+          <li><a href="./pages/gallery.html">Memes</a></li>
+          <li><a href="./pages/facts.html">Fun Facts</a></li>
```

Comment:
List items must be contained in an ordered or unordered list. 

When I [validate your HTML,](https://validator.w3.org/) I get this error:
<img width="951" alt="image" src="https://github.com/user-attachments/assets/c750d80b-b9b0-40b7-91e3-b5ec2a0a8193" />

```suggestion
        <ul>
          <li><a href="./pages/gallery.html">Memes</a></li>
          <li><a href="./pages/facts.html">Fun Facts</a></li>
        </ul>                  
```